src/components/reviewList.jsx

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+import { getReviews } from "../../utils";
+
+import "../App.css";
+
+export function GetReviews() {
+  const [reviews, setReviews] = useState([]);
+
```

Comment:
A state here for loading to provide the user with a visual indicator would be good here 🙂